packages/components/src/tabs/index.tsx

```diff
@@ -153,6 +153,14 @@ function Tabs( {
 		}
 	}, [ isControlled, selectedTab, selectedTabId, setSelectedId ] );
 
+	useEffect( () => {
+		// If there is no active tab, fallback to place focus on the first enabled tab
+		// so there is always an active element
+		if ( selectedTabId === null && ! activeId && firstEnabledTab?.id ) {
+			setActiveId( firstEnabledTab.id );
+		}
+	}, [ selectedTabId, activeId, firstEnabledTab?.id, setActiveId ] );
```

Comment:
I feel like it'd be safer to move this out to a designated effect like so, instead of piggybacking on another one. I'd want to avoid unintended complications down the line. In that vein, I think we'd be safer to add a `selectedTabId === null` condition, since that's the only time we need this logic, as far as we know. (And it doubles as a `isControlled` check.)

Does that sound reasonable? Anything I'm missing?

Reply:
I was considering this too from a code readability standpoint. I wasn't sure performance and debugging-wise if it would be worse to add another `useEffect` though. I'm totally fine with this route though if you are!